types/systemjs/index.d.ts

```diff
@@ -248,25 +263,16 @@ declare namespace SystemJSLoader {
         };
     }
 
-    interface SystemJSSystemFields {
```

Comment:
Why did you remove this interface?

Reply:
I removed it because it feels awkward. The _fields_ suffix is unusual, properties would be more standard but even then the segregation is not very unidiomatic. Overall I felt it would be more straightforward if the fields were inlined.